### PR TITLE
feat(phrases): add "ru" language

### PR DESCRIPTION
**Pull request by evist0**
## Summary
Added localization for Russian language.

## Testing
Locally built and launched the docker image. Checked that the translation appeared in sign-in experience.

### CODE REVIEW

File: packages/phrases-ui/src/locales/ru.ts [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    invalid_connector_auth: 'Авторизация недействительна',
+    invalid_connector_request: 'Данные коннектора недействительны.',
+    unknown: 'Неизвестная ошибка. Пожалуйста, повторите попытку позднее.',
+    invalid_session: 'Сессия не найдена. Пожалуйста, войдите снова.',
+  },
+};
+
+const ru = Object.freeze({
+  translation,
+});
```
Comment (charIeszhao) [inline packages/phrases-ui/src/locales/ru.ts:116]: Nit-picking: Should import and use a type here to make sure the phrase keys are identical to what we have in English phrase scripts.
```ts
const ru: LocalePhrase = Object.freeze(
  ...
)
```
But don't worry I can make that change on your behalf

**Review by charIeszhao — APPROVED** [review]
Overall looks good. Thanks for your contribution!

**Comment by charIeszhao** [comment]
Please contact me via discord (recommended) or email. We have a gift for you to claim

**Comment by evist0** [comment]
> Please contact me via [discord](https://discord.gg/hcW2xq2M) (recommended) or [email](mailto:charleszhao@silverhand.io). We have a gift for you to claim

I wasn't sure which text channel I should write to, so check your Discord direct messages.

**Pull request merged**